app/views/account_requests/new.html.erb

```diff
@@ -31,6 +31,7 @@
         </div>
 
         <div class="card-footer">
+          <%= recaptcha_tags %>
```

Comment:
Similarly to the comment above. Maybe we can conditionally render this depending on environment?

Reply:
Note: as mentioned in a comment in the commit I added, rendering (and verification) are controlled by the presence/absence of the env in `Recaptcha.configuration.skip_verify_env`, so there's no need for separate env guards around rendering/enforcing.